docs/docsite/rst/developers/contributing.rst

```diff
@@ -0,0 +1,467 @@
+              UI: http://localhost:3001
+     UI External: http://172.18.0.6:3001
+     -----------------------------------
+    [Browsersync] Serving files from: /galaxy
+    [03:10:04] Finished 'less' after 3.23 s
+    [03:10:04] Starting 'default'...
+    [03:10:04] Finished 'default' after 108 μs
+
+OK, go for it! Your Galaxy web site is available at: http://localhost:8000.
+
+.. note:: You won't be able to authenticate until you perform the post build 
+          steps for creating an `admin` user and configuring GitHub authentication.
+
+Post build setup
+----------------
+
+Create an admin user
+^^^^^^^^^^^^^^^^^^^^
+
+From the root of the project tree, run ``make dev/createsuperuser`` to start 
+the creation process. You'll be prompted for the vital details as depicted below:
+
+.. code-block:: shell-session
+
+    $ make dev/createsuperuser
+    Create Superuser
+
+    Username: admin
+    Email address: noemail@noemail.com
+    Password:
+    Password (again):
+    Superuser created successfully.
+
+The Django admin site can be accessed at http://localhost:8000/admin.
+
+Connect to GitHub
+-----------------
+
+To log into the development site, you first have to authorize it as a 
+GitHub Oauth Application. You can do this by logging into GitHub, 
+going to Personal Settings, choosing ``OAuth Applications``, 
+and then doing the following to create a new app:
+
+* Click **Register New Application**.
+* Set the *Homepage URL* to ``http://localhost:8000``.
+* Set the *Authorization Callback URL* to ``http://localhost:8000/accounts/github/login/callback/``.
+
+Log into your Galaxy admin site
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After you save the new application, access your local Galaxy admin site 
+at http://localhost:8000/admin, and log in using the admin user you created 
+above in [Create admin user](#create-admin-user)
+
+Update the site name
+^^^^^^^^^^^^^^^^^^^^
+
+Click on **Sites**. You'll see one site defined, ``example.com``. 
+Click on ``example.com`` to modify it. On the next page, change both the 
+**Domain Name** and **Display Name** from ``example.com`` to ``localhost``. 
+Click the **Save** button.
+
+Create a new social application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Next, create a new social application. Start by finding ``Social applications``
+at the bottom of the list, on the admin site home page. 
+Click the **Add** button to its right, and on the next page, complete the
+following to configure the new application:
+
+* Set the **Provider** to ``GitHub``.
+* Enter ``GitHub`` as the **Name**.
+* From the new GitHub OAuth application you just created, copy the 
+  **ClientID** value into **Client id**.
+* Copy the **Client Secret** value into **Secret key**.
+* Under **Sites**, add ``localhost`` to **Chosen sites**. Save the changes.
+
+Now test the authentication. Log out of your admin account, and go back to the 
+home page at http://localhost:8000. Now log in using your GitHub account by 
```

Comment:
Highlight the URL with double ticks, to be consistent with other links.

Reply:
Done.